packages/seo-integration/src/replacement-variables.js

```diff
@@ -0,0 +1,111 @@
+import { select } from "@wordpress/data";
+import { addFilter, removeFilter } from "@wordpress/hooks";
+import { __ } from "@wordpress/i18n";
+import createReplacementVariables from "@yoast/replacement-variables";
+import { SEO_STORE_NAME } from "@yoast/seo-store";
+import { curryRight, mapValues, noop, reduce, unionBy } from "lodash";
+
+/**
+ * Creates the default replacement variable configurations, for use within the SEO store context.
+ *
+ * There are more replacement variables outside of the SEO store context.
+ * @see [Available replacement variables in Yoast SEO for WordPress]{@link https://yoast.com/help/list-available-snippet-variables-yoast-seo/}
+ *
+ * @returns {Object.<string, ReplacementVariableConfiguration>} The default replacement variable configurations, keyed by name for easy reference.
+ */
+export const createDefaultReplacementVariableConfigurations = () => ( {
+	title: {
+		name: "title",
+		label: __( "Title", "wordpress-seo" ),
+		getReplacement: () => select( SEO_STORE_NAME ).selectTitle(),
+	},
+	excerpt: {
+		name: "excerpt",
+		label: __( "Excerpt", "wordpress-seo" ),
+		getReplacement: () => select( SEO_STORE_NAME ).selectExcerpt(),
+	},
+	date: {
+		name: "date",
+		label: __( "Date", "wordpress-seo" ),
+		getReplacement: () => select( SEO_STORE_NAME ).selectDate(),
+	},
+	focusKeyphrase: {
+		name: "focusKeyphrase",
+		label: __( "Focus keyphrase", "wordpress-seo" ),
+		getReplacement: () => select( SEO_STORE_NAME ).selectKeyphrase(),
+		regexp: new RegExp( "%%focuskw%%|%%keyword%%", "g" ),
+	},
+} );
+
+/**
+ * Merges two sets of replacement variable configurations, by name.
+ *
+ * @param {ReplacementVariableConfiguration[]} default Default set. An entry will be overriden whenever its name is not unique.
+ * @param {ReplacementVariableConfiguration[]} preferred Preferred set. Possibly overrides the default set.
+ *
+ * @returns {ReplacementVariableConfiguration[]} Replacement variable configurations.
+ */
+export const mergeReplacementVariableConfigurations = curryRight( unionBy )( "name" );
```

Comment:
I'm not seeing this function actively being used right now.

Reply:
This was a helper for the implementer, but with the change of the defaults to an Object, and the scope I should just remove this 👍